envFiles/.env.devcontainer

```
@@ -31,8 +31,12 @@ API_JWT_SECRET=b4896453be722d5ca94058a73f52b31c75980b485fa6d74d91f417a8059d8731
 API_COOKIE_SECRET=b4896453be722d5ca94058a73f52b31c75980b485fa6d74d91f417a8059d8731
 # REST auth (required when NODE_ENV=production): JWT secret for access/refresh tokens (jose).
 API_AUTH_JWT_SECRET=b4896453be722d5ca94058a73f52b31c75980b485fa6d74d91f417a8059d8731
-# API_ACCESS_TOKEN_TTL=900
-# API_REFRESH_TOKEN_TTL=2592000
+API_ACCESS_TOKEN_TTL=900
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_ACCESS_TOKEN_TTL key should go before the API_ADMINISTRATOR_USER_EMAIL_ADDRESS key

(UnorderedKey)